source/macros.tex

```diff
@@ -507,3 +507,4 @@
 \subsubsection[#1]{\hfill[#2]}\vspace{-.3\onelineskip}\label{#2}\textbf{#1}\\%
 }
 \newcommand{\defncontext}[1]{\textlangle#1\textrangle}
+\newcommand{\defnnote}[2]{{\small Note #1 to entry: #2}}
```

Comment:
We had discussed this, and as much as we all love ISO drafting directive normative annex D, zygoloid called this "overreaching" if I recall correctly.